### DISCUSSION TITLE
When will it leave Beta status?

### DISCUSSION THREAD

**Discussion by Primajin** [clauderic/dnd-kit#418] [General]
We are highly interested in this project, is there already a timeline / plan when it will leave the beta phase?

Thanks and all the best!

**Comment by clauderic** [comment]
This project is in beta because there are currently some features that are not well supported. Having said that, this project is using semantic versioning, so major breaking changes will be identified and have clear migration steps outlined.

If the use case you're planning to build is similar to any of the examples on https://examples.dndkit.com, chances are that your use-case is well supported and production-ready.

We use @dnd-kit at scale at Shopify for millions of users, so you should feel fairly confident that it is a production-ready library, as long as your use-case is similar to one of the documented examples.

Examples of not well supported use-cases include [fixed position droppable regions](https://github.com/clauderic/dnd-kit/issues/298) and [fixed position scroll containers](https://github.com/clauderic/dnd-kit/issues/61), and there are also a couple bugs with [collision detection when scrolling a list of droppable elements and trying to drag over a different droppable container that is not within the scrollable list](https://github.com/clauderic/dnd-kit/issues/43).

All of these issues are solvable given enough time and effort. Some of them will require breaking changes in the API, which is another reason why the library is labeled as beta. The core concepts will remain, but it's possible that you'll have to make some updates to you application if and when breaking changes are introduced. This project follows semantic versioning, so breaking changes will be clearly labeled as such and have migration steps outlined.

**Comment by Primajin** [comment]
Great! We moved from react-sortable-hoc to @dnd-kit 🎉

**Comment by acherkashin** [comment]
@clauderic thank you for this great library, I really enjoyed playing with the use cases in storybook 👍 

Don't you mind telling me when the library will leave the beta phase?
